packages/template-retail-react-app/app/commerce-api/index.js

```diff
@@ -112,72 +113,91 @@ class CommerceAPI {
             }
         }
 
-        /** ************ Commerce SDK React Clients ************ */
-
-        this._reactSdkConfig = {
-            proxy,
-            parameters: this._config.parameters,
-            headers: this._config.headers
-        }
-
-        this._reactSdkClients = {
-            shopperLogin: new sdk.ShopperLogin(this._reactSdkConfig)
-        }
-
-        /** ************ End Commerce SDK React Clients ************ */
-
         // Instantiate the SDK class proxies and create getters from our api mapping.
         // The proxy handlers are called when accessing any of the mapped SDK class
         // proxies, executing various pre-defined hooks for tapping into or modifying
         // the outgoing method parameters and/or incoming SDK responses
```

Comment:
Nit: The implentation has become more obstract, can we edit the comment to explain what it does here?